Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -605,7 +621,8 @@ def audit_revision_and_version_scheme
       next unless spec = formula.send(spec_sym)
       next unless previous_version_and_checksum[spec_sym][:version] == spec.version
       next if previous_version_and_checksum[spec_sym][:checksum] == spec.checksum
-      problem "#{spec_sym}: sha256 changed without the version also changing; please create an issue upstream to rule out malicious circumstances and to find out why the file changed."
+      problem "#{spec_sym}: sha256 changed without the version also changing; please create an issue upstream to rule out malicious circumstances and to find out why the file changed.",
```

Comment:
Everything in `audit_revision_and_version_scheme` is _never run_ for new formulae.